client/src/components/LandingPage/AniCard.js

```diff
@@ -61,6 +61,9 @@ export default function AniCard({ number }) {
         <Button
           size="small"
           className={classes.adjust}
+          style={{backgroundColor:"#f26522",
+        borderRadius:"999999999999999999999999999px"
```

Comment:
Why is this value so large?

Reply:
it's a trick to make a perfect pill shape button.So i chose a large value.